Invalid signature error

Might be a side effect of recent merges, should find out where.
```
2018/09/11 15:25:14.701538 [INFO] > made SH script, state 0
2018/09/11 15:25:14.701555 [INFO] 	 revPub 02c5c1167148f232dabef81863c345b454fdaa4bcf9aca9e944cc6e04aba0422b4 timeout pub 02e91cc8c76be3d178c8e4433c7a3fcf95c9c6729ca9792f32a849bc8381dcef48
2018/09/11 15:25:14.701562 [INFO] 	 script 632102c5c1167148f232dabef81863c345b454fdaa4bcf9aca9e944cc6e04aba0422b46755b2752102e91cc8c76be3d178c8e4433c7a3fcf95c9c6729ca9792f32a849bc8381dcef4868ac
2018/09/11 15:25:14.701570 [INFO] 	 scripthash 0020e4b26b9b6df2c2912fe7e2b8727eeaf963cff7e0ca48becd4223f5ad4aeda3da
	combined refund 02e04b65358bb5e121aff550d224a57449414d0b15448c649efb670d3b87cf09c1, pkh 0014115421b88a786236a38e72596daf7e6ea9f172de, amt 120000
2018/09/11 15:25:14.701616 [INFO] Verifying signatures with Elk [031e822c8d511f60ec9aaa3100882442e822d2acd6714abdba6956ebfc0f0983b7] NextElk [0366ae6af5c44a567c9858433c33ea9e59ac9603f1c327d5350ec70310f94170e0] N2Elk [02290990fad6466b0b50d11b74880693de9a0b5d81e234439f5c13548537ca34b3]
____ sig verification for channel (2,4):
	input c067303986b5e3c9dac8d2d9ba67964ef248771d34a3976b05e6715b3b5dd40d;0
	output 0: 0014115421b88a786236a38e72596daf7e6ea9f172de 120000
	output 1: 0020e4b26b9b6df2c2912fe7e2b8727eeaf963cff7e0ca48becd4223f5ad4aeda3da 720000
2018/09/11 15:25:14.701928 [INFO] 	state 0 myamt: 800000 theiramt: 200000
2018/09/11 15:25:14.701942 [INFO] 	sig: 69fb1111c1e3080d8ef5d7ee2b9b46f51d3208a0c385f0c4b4e83eb17c4f75d035e47c8e3468fd1ea2b3ad8471a3d77be329a458f6d5fc52b122631339f789e0
2018/09/11 15:25:14.702588 [ERROR] QChanAckHandler VerifySig err Invalid signature on chan 4 state 0
2018/09/11 15:25:16.393902 [INFO] decrypted message is 70e73e6029b197f324da32cddc7873a5b03259b7a4e7d1066c27b86518342363fcf7d253bd0bec78d40000000000030e7a975aa28a55f82b1b7cc46429921bd80ea9a0cf280000010100000002
```

--- a/qln/dualfund.go
+++ b/qln/dualfund.go
@@ -441,7 +441,7 @@ func (nd *LitNode) DualFundingAcceptHandler(msg lnutil.DualFundingAcceptMsg) {
 	q.State.MyAmt = nd.InProgDual.OurAmount
 	// get fee from sub wallet.  Later should make fee per channel and update state
 	// based on size
-	q.State.Fee = nd.SubWallet[q.Coin()].Fee() * 1000
+	q.State.Fee = nd.SubWallet[q.Coin()].Fee() * consts.QcStateFee
 	q.Value = nd.InProgDual.OurAmount + nd.InProgDual.TheirAmount
 
 	q.State.NextHTLCBase = msg.OurNextHTLCBase

--- a/qln/fund.go
+++ b/qln/fund.go
@@ -346,7 +346,7 @@ func (nd *LitNode) PointRespHandler(msg lnutil.PointRespMsg) error {
 	q.State.MyAmt = nd.InProg.Amt - nd.InProg.InitSend
 	// get fee from sub wallet.  Later should make fee per channel and update state
 	// based on size
-	q.State.Fee = nd.SubWallet[q.Coin()].Fee() * 1000
+	q.State.Fee = nd.SubWallet[q.Coin()].Fee() * consts.QcStateFee
 
 	q.State.Data = nd.InProg.Data
 

